scripts/bulk-mailer/send-email-batch.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const P = require('../../lib/promise')
+
+module.exports = function sendBatch(batch, sendEmail, log) {
+  let successCount = 0
+  let errorCount = 0
+
+  return P.all(
+    batch.map(userRecord => {
+      return sendEmail(userRecord)
+        .then(
+          () => {
+            successCount++
+            log.info({
```

Comment:
`log.info` here must be a no-op here as I do not see this message (although if I stick a `console.log` right there, that is printed).

Reply:
Fixed